src/app/locale/getDefaultTranslations.ts

```diff
@@ -3,55 +3,60 @@ import { Translations } from '@bigcommerce/checkout-sdk';
-                './translations/de.json'
-            )).default);
-
-        case 'fr':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-fr" */
-                './translations/fr.json'
-            )).default);
-
-        case 'it':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-it" */
-                './translations/it.json'
-            )).default);
-
-        case 'nl':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-nl" */
-                './translations/nl.json'
-            )).default);
-
-        case 'pt-BR':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-pt-br" */
-                './translations/pt-BR.json'
-            )).default);
-
-        case 'pt':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-pt" */
-                './translations/pt.json'
-            )).default);
-
-        case 'sv':
-            return asTranslations((await import(
-                /* webpackChunkName: "translations-sv" */
-                './translations/sv.json'
-            )).default);
-
-        case 'en':
-            return FALLBACK_TRANSLATIONS;
-
-        default:
-            return {};
+    if (matchLocale(locale, 'de')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-de" */
+            './translations/de.json'
+        )).default);
     }
+
+    if (matchLocale(locale, 'fr')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-fr" */
+            './translations/fr.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'it')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-it" */
+            './translations/it.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'nl')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-nl" */
+            './translations/nl.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'pt-BR')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-pt-br" */
+            './translations/pt-BR.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'pt')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-pt" */
+            './translations/pt.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'sv')) {
+        return asTranslations((await import(
+            /* webpackChunkName: "translations-sv" */
+            './translations/sv.json'
+        )).default);
+    }
+
+    if (matchLocale(locale, 'en')) {
+        return FALLBACK_TRANSLATIONS;
+    }
+
+    return {};
```

Comment:
🤔  No default translation?

Reply:
Yes, so I made a distinction between default translations and fallback translations. Fallback translations always exist and in English. Default translations may not exist if there are no default translations matching the specified locale.